lib/enumerations/base.rb

```diff
@@ -4,7 +4,7 @@
 require 'enumerations/finder_methods'
 
 module Enumerations
-  class Base
+  class Base < ActiveSupport::Multibyte::Chars
```

Comment:
It's ok, but you should add `super(symbol)` into #initialize to avoid warnings:
`/Users/petarcurkovic/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/activesupport-5.2.1/lib/active_support/multibyte/chars.rb:74: warning: instance variable @wrapped_string not initialized`

Reply:
Yes true, nice catch 👍